plugins/TagFix_Opening_Hours.py

```diff
@@ -42,14 +42,14 @@ def sanitize_tags(self, tags):
         if 'opening_hours' not in tags:
             return
 
-        try:
-            sanitized_field = sanitize_field(tags['opening_hours'])
-            if sanitized_field.replace(' ', '').replace('0', '').lower() != tags['opening_hours'].replace(' ', '').replace('0', '').lower(): # Ignore sapce and 0 changes
-                return {"class": 32501, "subclass": 0, 'fix': {'opening_hours': sanitized_field}}
-        except InconsistentField as e:
-            return {"class": 32501, "subclass": 1, 'text': {'en': str(e)}}
-        except SanitizeError:
-            return {"class": 32501, "subclass": 2}
+        parser = OpeningHours()
+        parser.setExpression(tags['opening_hours'])
+        if parser.error() != Error.NoError:
```

Comment:
I think this should check for Error.SyntaxError || Error.IncompatibleMode rather than NoError. This will fix false-positives e.g. for Error.UnsupportedFeature which only matters for evaluation not for validation (see e.g. the reports for "12:00+" or "SH" selectors, both perfectly valid).

Reply:
Thanks, done.